tark/tark_web/templates/search_result.html

```diff
@@ -374,7 +374,8 @@
             });          
         },
      
-    "order": [[ 11, "desc" ], [ 4, "desc" ], [ 6, "desc" ]],
+    /* change order to Type desc,  Source asc, Strand desc, Assembly desc, From release desc*/
+    "order": [[ 14, "desc" ], [ 5, "asc" ], [ 11, "desc" ], [ 4, "desc" ], [ 6, "desc" ]],
```

Comment:
Just out of curiosity, what do these numbers mean (14, 5, etc)?

Reply:
@bilalebi, These numbers are column indexes, starting from 0 for the first column

Reply:
I see, thank you